mods/cnc/chrome/editor.yaml

```diff
@@ -584,19 +584,8 @@ Container@EDITOR_WORLD_ROOT:
 					Height: 25
 					Text: History
 					Font: Bold
-		Button@BUILDABLE_BUTTON:
-			X: WINDOW_RIGHT - 1020
-			Y: 5
-			Width: 100
-			Height: 25
-			Text: Buildable
-			Font: Bold
-			Key: f2
```

Comment:
I don't reimplement hotkeys in this pr. They need to be handled differently not just because of the dropdown works, but also because they aren't editable by users in the hotkeys menu

Reply:
I added the hotkey just because everything else had one, but I don't think this requires a quick keyboard access.

Reply:
I think having those on hotkeys could lead to a smoother workflow. Regardless I believe we should have a dropdown class and unhardcode values into yaml. That could also allow for a smoother reimplementation of hotkeys

Reply:
but that would be worthy of a separate pr